walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/Event.java

```diff
@@ -93,4 +95,15 @@ private static boolean equals(Event a, Event b) {
     // interface documentation, equals will check for set-equality across all set implementations.
     return a.title.equals(b.title) && a.when.equals(b.when) && a.attendees.equals(b.attendees);
   }
+
+  public static final Comparator<Event> EVENT_COMPARATOR = new Comparator<Event>() {
+
+    @Override
+    public int compare(Event a, Event b) {
+      if (a.getWhen().start() != b.getWhen().start())
+        return TimeRange.ORDER_BY_START.compare(a.getWhen(), b.getWhen());
+      else
+        return TimeRange.ORDER_BY_END.compare(a.getWhen(), b.getWhen());
+    }
+  };
```

Comment:
After searching for the comparable interface I think it should stay as a comparator because the comparable interface works and affects the original object because it uses this. Originally it was because I based it of https://github.com/googleinterns/step/blob/0218d07ca16ad8ead819a051f69edb563d030f86/walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/TimeRange.java#L32.

Reply:
Ok, makes sense!